### PR TITLE
clean up metricRegistry in close()

### PR DESCRIPTION
**Pull request by mberndt123**
Hi,

I'm using your project in a Play-based web app. In Play's dev mode I keep getting warnings like these:
```metricChange called for `org.apache.kafka.common.metrics.consumer-coordinator-metrics.consumer-config-site.last-heartbeat-seconds-ago' which was already registered, ignoring.```
This is due to the fact that Play's dev mode reloads the application without restarting the JVM, so the metrics registry isn't cleaned up. Hence the PR, please consider merging it.

All the best

### CODE REVIEW

File: src/main/java/com/simple/metrics/kafka/DropwizardReporter.java [hunk 1]
```diff
@@ -61,10 +65,14 @@ public void metricRemoval(KafkaMetric kafkaMetric) {
...
         LOGGER.debug("Removing {}", name);
         registry.remove(name);
+        metricNames.remove(name);
     }
 
     @Override
-    public void close() {}
+    public void close() {
+        for (String name: metricNames)
+            registry.remove(name);
```
Comment (jklukas) [inline src/main/java/com/simple/metrics/kafka/DropwizardReporter.java]: Looks like [MetricRegistry.remove](http://metrics.dropwizard.io/3.1.0/apidocs/com/codahale/metrics/MetricRegistry.html#remove-java.lang.String-) is safe to call again for a metric that's already been removed, so this should be fine.

File: src/main/java/com/simple/metrics/kafka/DropwizardReporter.java [hunk 2]
```diff
@@ -61,10 +65,14 @@ public void metricRemoval(KafkaMetric kafkaMetric) {
         String name = dropwizardMetricName(kafkaMetric);
         LOGGER.debug("Removing {}", name);
         registry.remove(name);
+        metricNames.remove(name);
```
Comment (jklukas) [inline src/main/java/com/simple/metrics/kafka/DropwizardReporter.java]: Looks like [HashSet.remove](https://docs.oracle.com/javase/7/docs/api/java/util/HashSet.html#remove(java.lang.Object)) is safe to call on a metric that's already been removed, so this should be fine.

**Comment by mberndt123** [comment]
Cool

**Comment by jklukas** [comment]
I rebased on master to pull in #6 and force pushed the result. I'm going to merge and cut a 1.1.0 release.

**Comment by jklukas** [comment]
1.1.0 is published and available: https://search.maven.org/#artifactdetails%7Ccom.simple%7Ckafka-dropwizard-reporter%7C1.1.0%7Cjar

**Comment by mberndt123** [comment]
Thanks for the fast release

**Pull request merged**